src/hotspot/share/opto/loopopts.cpp

```diff
@@ -2159,7 +2164,7 @@ void PhaseIdealLoop::clone_loop_handle_data_uses(Node* old, Node_List &old_new,
       // make sure the Bool/Cmp input is cloned down to avoid a Phi between
       // the AllocateArray node and its ValidLengthTest input that could cause
       // split if to break.
-      if (use->is_If() || use->is_CMove() || use->Opcode() == Op_Opaque4 ||
+      if (use->is_If() || use->is_CMove() || use->is_Opaque4() ||
```

Comment:
Added case for Initialized Assertion Predicate. Covered by `testOpaqueOutsideLoop()`.